src/traces/cone/convert.js

```diff
@@ -88,15 +87,21 @@ function convert(scene, trace) {
 
     coneOpts.colormap = parseColorScale(trace.colorscale);
     coneOpts.vertexIntensityBounds = [trace.cmin / trace._normMax, trace.cmax / trace._normMax];
-
-    coneOpts[sizeMode2sizeKey[trace.sizemode]] = trace.sizeref;
     coneOpts.coneOffset = anchor2coneOffset[trace.anchor];
 
-    var meshData = conePlot(coneOpts);
+    if(trace.sizemode === 'scaled') {
+        // unitless sizeref
+        coneOpts.coneSize = trace.sizeref || 0.5;
+    } else {
+        // sizeref here has unit of velocity
+        coneOpts.coneSize = (trace.sizeref / trace._normMax) || 0.5;
```

Comment:
Previously, `coneSize` under `sizemode: 'absolute'` was scaled using the max norm of the "scaled" u/v/w vectors (that same `dataScale` than in https://github.com/plotly/plotly.js/issues/2646). This made `sizeref` under `sizemode: 'absolute'` had little meaning. 

Now, `sizeref` under `sizemode: 'absolute' is scaled by the "input" u/v/w max norm should make this thing a little less confusing.

Reply:
> What if the field is uniformly zero? 

Oh right, `Infinity` is truthy. Good call!

Reply:
> When is the || 0.5 (in either case) necessary?

Currently, the `sizeref` attribute declaration does not set a `dflt` field. In practice I guess it does have a hard default of `0.5`. But technically under the `sizemode: 'absolute'`, the sizemode default is `0.5 * trace._normMax`.